pydatalab/pydatalab/models/people.py

```diff
@@ -62,6 +62,17 @@ def add_missing_verification(cls, v):
         return v
 
 
+class DisplayName(str):
+    """A constrained string less than 150 characters long."""
+
+    max_length = 150
+
+    def __new__(cls, value):
+        if len(value) > cls.max_length:
+            raise ValueError(f"Display name must be at most {cls.max_length} characters long.")
+        return str.__new__(cls, value)
```

Comment:
We should also check for emptiness, and that all characters are in an alphabet.

Could you add some tests in the pure Python for this? e.g., passing a series of valid names and testing that they work, and a series of invalid names and checking that they don't? I can help talk you through the test set up process if need be